Cython/Utility/ImportExport.c

```diff
@@ -415,11 +415,26 @@ static int __Pyx_SetPackagePathFromImportLib(const char* parent_package_name, Py
 //@substitute: naming
 
 // PY_VERSION_HEX >= 0x03030000
-#if PY_MAJOR_VERSION >= 3 && !CYTHON_PEP489_MULTI_PHASE_INIT
+#if !CYTHON_PEP489_MULTI_PHASE_INIT
 static int __Pyx_SetPackagePathFromImportLib(const char* parent_package_name, PyObject *module_name) {
-    PyObject *importlib, *loader, *osmod, *ossep, *parts, *package_path;
-    PyObject *path = NULL, *file_path = NULL;
+    PyObject *importlib, *osmod, *ossep, *parts, *package_path;
+    PyObject *file_path = NULL;
     int result;
+#if PY_VERSION_HEX > 0x03040000
```

Comment:
`>=`? I think the function was added in 3.4

Reply:
ok